zerver/lib/notifications.py

```diff
@@ -302,8 +298,10 @@ def do_send_missedmessage_events_reply_in_zulip(user_profile, missed_messages, m
         else:
             huddle_display_name = u"%s, and %s others" % (', '.join(other_recipients[:2]), len(other_recipients) - 2)
             context.update({'huddle_display_name': huddle_display_name})
+        reply_to_name = huddle_display_name
     elif (missed_messages[0].recipient.type == Recipient.PERSONAL):
         context.update({'private_message': True})
+        reply_to_name = cast(Text, missed_messages[0].sender.full_name)
```

Comment:
Looking at https://code.djangoproject.com/ticket/25986 and the upstream issue http://bugs.python.org/issue25955, it seems that formataddr is deprecated and doesn't support RFC 6532.  I think passing in the `from_name` and `from_address` as a tuple into the `EmailMultiAlternatives` constructor should work, since it looks like Django (since 1.10) handles this.